docs/deployment-and-operations/packaging.md

```diff
@@ -60,12 +60,19 @@ If you prefer to package your job to submit to an existing Flink cluster, simply
 {% endhighlight %}
 
 It includes all of Stateful Functions' runtime dependencies and configures the application's main entry-point.
-You do not need to take any action beyond adding the dependency to your POM.
 
 <div class="alert alert-info">
   <strong>Attention:</strong> The distribution must be bundled in your application fat JAR so that it is on Flink's <a href="https://ci.apache.org/projects/flink/flink-docs-stable/monitoring/debugging_classloading.html#inverted-class-loading-and-classloader-resolution-order">user code class loader</a>
 </div>
 
 {% highlight bash %}
-./bin/flink run ./statefun-example.jar
+./bin/flink run -c org.apache.flink.statefun.flink.core.StatefulFunctionsJob ./statefun-example.jar
 {% endhighlight %}
+
+You will also need to add the following two configurations to your clusters `flink-conf.yaml`.
```

Comment:
Could we rephrase this as:
these configurations are strictly required for running Statefun applications, or something along those lines?

As such, the max concurrent checkpoints should also be mentioned here.